infrastructure/api/api_test.go

```diff
@@ -0,0 +1,85 @@
+package api
+
+import (
+	"testing"
+
+	"github.com/OrlandoRomo/academy-go-q32021/domain/model"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	testApiKey = "cc2464e4a1mshb5ceeca91e5a6adp1fa80bjsn4b48e2408b87"
+)
+
+func TestUpdateCSVPath(t *testing.T) {
+	t.Run("set valid CSV path", func(t *testing.T) {
+		urbanClient := NewUrbanDictionary(testApiKey)
+		urbanClient.UpdateCSVPath("/usr/local/definitions.csv")
+		assert.Equal(t, urbanClient.CSVPath, "/usr/local/definitions.csv")
+	})
+}
+
+func TestGetDefinitions(t *testing.T) {
+	urbanClint := NewUrbanDictionary(testApiKey)
+	t.Run("success - definition by term", func(t *testing.T) {
+		definitions, err := urbanClint.GetDefinitions("ha")
```

Comment:
I suggest taking a look at the article I shared previously to mock the http calls, you don't want to call the actual http server on tests. You'll need to refactor a bit your api service to be able to mock the http client